### PR TITLE
Fix curriculum guide

### PR DESCRIPTION
**Pull request by kaleemullah786**
kindly review the tabs added to curriculum-guides

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "tailwindcss": "^3.2.7"
   },
   "dependencies": {
-    "file-system": "^2.2.2",
+    "file-system": "^1.2.2",
```
[the remark this answers is not in the document]
Comment (kaleemullah786) [reply package.json:25]: No, you can change back to the older one. I guess npm just updated it to latest version while installing dependencies

File: pages/requirements/curriculum-guide.js [hunk 2]
```diff
@@ -21,17 +27,17 @@ const CurriculumGuide = ({ curriculumGuides }) => {
         <title>چارت درسی | انجمن کامپیوتر صنعتی ارومیه</title>
       </Head>
       <h2>چارت</h2>
-      {curriculumGuides.map((curriculumGuide, curriculumGuideIndex) => (
```
Comment (amirihusayn) [inline pages/requirements/curriculum-guide.js:24]: The reason We map through `curriculumGuides` is that there will be several of them in future; But currently there is just one.  
Also notice in line 14 `curriculumGuides[0]`. As I mentioned may be several `curriculumGuide` inside `curriculumGuides` array.

**Comment by amirihusayn** [comment]
```jsx
      {curriculumGuides.map((curriculumGuide, curriculumGuideIndex) => (
        <div key={curriculumGuideIndex}>
          <TableFromArray
            array={curriculumGuide.contents[0]}
            comments={curriculumGuide.props[0].comments}
          />
          <Tabs headers={curriculumGuide.props.slice(1).map(prop => prop.comments[0])}
          contents={curriculumGuide.contents.slice(1).map((semester, semesterIndex) => 
          (<TableFromArray array={semester} comments={curriculumGuide.props.slice(1)[semesterIndex].comments} key={semesterIndex} />))} />
        </div>
      ))}
```

**Comment by kaleemullah786** [comment]
kindly review the changes

**Pull request merged**